### ISSUE TITLE
Metrics/Profile: "Latest Updates" plot label colors are illegible and flash when hovering

### ISSUE THREAD

**Issue by mburrus** [NCEAS/metacatui#1545]
**Describe the bug**
When on a data repository's profile page (or the repository's summary page), the "Latest Updates" plot label appears when hovering over a plot point, however, it has a dark navy background and dark mustard text color, making it very hard to read. Also, depending on your placement of the mouse while hovering, the label may flash aggressively. 

Note this is not an issue for customizable data portal metrics.

**To Reproduce**
1. Go to https://data.ess-dive.lbl.gov/profile or https://arcticdata.io/catalog/profile  
2. Scroll down to "Latest Updates" graph
3. Hover mouse over any plot point

**Expected behavior**
When hovering over a plot point in the "Latest Updates" plot, I expect to see a static label in some combination of contrasting colors (I think it used to be navy blue and white for a time?). 

**Screenshots**
<img width="670" alt="Screen Shot 2020-10-19 at 12 10 00 PM" src="https://user-images.githubusercontent.com/54084565/96477082-ebbd4000-11ea-11eb-98f5-b05f2f251ae7.png">

**Desktop:**
 - OS: macOS Catalina 10.15.6
 - Browser: Chrome 
 - Version: Version 86.0.4240.80

**Comment by mbjones** [comment]
I can see this problem on Firefox 81.0 as well, so it is not browser-specific.

**Comment by laurenwalker** [comment]
Thanks, I will take a look at this

**Comment by laurenwalker** [comment]
I've fixed this by increasing the size of each data point when you are hovering on it, so it's harder to mouseout. I also added a quick delay to hiding the label so it's still visible for 300 milliseconds after leaving the data  point.

![fix.gif](https://images.zenhubusercontent.com/5d152e12c27d0c306f021863/26aed01e-4b1a-4ee2-a4e2-616bc5179f2b)

The text color was coming from all the theme CSS files, so I removed those. You'll want to make sure your ESS-DIVE theme doesn't have that same CSS rule in your theme CSS. (`.line-chart-label.text`)
